roles/openshift_cli/tasks/main.yml

```diff
@@ -3,34 +3,18 @@
   package:
     name: "{{ openshift_service_type }}-clients{{ openshift_pkg_version | default('') }}"
     state: present
-  when: not openshift_is_atomic | bool
   register: result
   until: result is succeeded
 
 - block:
-  - name: Pull CLI Image (docker)
+  - name: Pull CLI Image
```

Comment:
Shortly after this pull merged, the openshift extended_install job started failing and blocking merges to origin:

https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_extended_conformance_install/16436/consoleFull#181091684258b6e51eb7608a5981914356

```

TASK [openshift_cli : Install clients] *****************************************
task path: /usr/share/ansible/openshift-ansible/roles/openshift_cli/tasks/main.yml:2
ok: [localhost] => {
    "attempts": 1, 
    "changed": false, 
    "generated_timestamp": "2018-10-25 13:23:31.669277", 
    "msg": "", 
    "rc": 0, 
    "results": [
        "origin-clients-4.0.0-0.alpha.0.409.853d900.x86_64 providing origin-clients-4.0.0-0.alpha.0.409.853d900 is already installed"
    ]
}

TASK [openshift_cli : Pull CLI Image] ******************************************
task path: /usr/share/ansible/openshift-ansible/roles/openshift_cli/tasks/main.yml:10
fatal: [localhost]: FAILED! => {
    "changed": false, 
    "generated_timestamp": "2018-10-25 13:23:32.405818", 
    "msg": "Error pulling docker.io/openshift/origin-node - code: None message: manifest for docker.io/openshift/origin-node:853d900 not found"
}

PLAY RECAP *********************************************************************
 [WARNING]: Could not create retry file '/usr/share/ansible/openshift-
ansible/playbooks/deploy_cluster.retry'.         [Errno 13] Permission denied:
u'/usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.retry'
localhost                  : ok=228  changed=93   unreachable=0    failed=1   
```

That seems like too much of a coincidence.  Did the default change so that this block was getting called?

Reply:
I guess we've already removed that job, so we can simply turn it off.